### ISSUE TITLE
Can not use woodstox SAX Parser with JAXB due to unrecognized feature

### ISSUE THREAD

**Issue by PascalSchumacher** [FasterXML/woodstox#77]
It currently not possible to use the Woodstox SAX Parser with JAXB (2.3.0+).

I'm using `com.sun.xml.bind:jaxb-impl:2.3.2`. When I set the `javax.xml.parsers.SAXParserFactory=com.ctc.wstx.sax.WstxSAXParserFactory` system property unmarshalling fails with:

```
Juni 12, 2019 9:58:03 VORM. com.sun.xml.bind.v2.util.XmlFactory createParserFactory
SEVERE: null
org.xml.sax.SAXNotRecognizedException: Feature 'http://javax.xml.XMLConstants/feature/secure-processing' not recognized
	at com.ctc.wstx.sax.WstxSAXParserFactory.setFeature(WstxSAXParserFactory.java:154)
	at com.sun.xml.bind.v2.util.XmlFactory.createParserFactory(XmlFactory.java:106)
	at com.sun.xml.bind.v2.runtime.unmarshaller.UnmarshallerImpl.getXMLReader(UnmarshallerImpl.java:124)
	at javax.xml.bind.helpers.AbstractUnmarshallerImpl.unmarshal(AbstractUnmarshallerImpl.java:170)
	at javax.xml.bind.helpers.AbstractUnmarshallerImpl.unmarshal(AbstractUnmarshallerImpl.java:219)
	...
```

I guess this issue would be solved if https://github.com/FasterXML/woodstox/issues/61 is implemented.

By the way: Thank you very much for providing woodstox.

**Comment by cowtowncoder** [comment]
Fixed via #61, will be in 5.3.0 which I hope to release soon

**Comment by PascalSchumacher** [comment]
Thank you very much! 👍 

I am looking forward to the release.
